docs/ref/models/querysets.txt

```diff
@@ -2666,6 +2666,12 @@ Note the first query will match ``'Beatles Blog'``, ``'beatles blog'``,
     :ref:`database note <sqlite-string-matching>` about string comparisons.
     SQLite does not do case-insensitive matching for non-ASCII strings.
 
+.. admonition:: ``UUIDField`` on PostgreSQL
```

Comment:
I think these admonitions are in the wrong place... We adding a lot of noise (one identical admonition for each lookup) in the main docs (the lookup docs!!!) for something that's a mega-edge-case™. 

I would place a single admonition in the UUIDField ref. Most users won't need it, but we're not forcing it upon people repeatedly in lots of places if we put it there.